recipe/meta.yaml

```diff
@@ -15,14 +15,18 @@ build:
 
 requirements:
   host:
-    - python
+    - python >=3.6
     - pip
   run:
-    - python
+    - python >=3.6
```

Comment:
From upstream:
```
install_requires=["colorama; platform_system == 'Windows'"]
```
@ocefpaf and @pmlandwehr, how do we want to handle this?  `colorama` for all platforms?  Removing `noarch: python`?

Reply:
I think I'm going to trust upstream and remove `noarch: python` so we can just add that dependency for `win`.  But let me know if you disagree and we can go back.

Reply:
Yeah, that is the way to fix this at the moment. Conda is not the best tool when it comes to "some arch."